src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -158,12 +164,15 @@ public function load(array $configs, ContainerBuilder $container)
             $this->registerRequestConfiguration($config['request'], $container, $loader);
         }
 
+        $container->addResource(new ClassExistenceResource(Validation::class));
+        $container->addResource(new ClassExistenceResource(Form::class));
+
         if ($this->isConfigEnabled($container, $config['form'])) {
             $this->formConfigEnabled = true;
             $this->registerFormConfiguration($config, $container, $loader);
             $config['validation']['enabled'] = true;
 
-            if (!class_exists('Symfony\Component\Validator\Validation')) {
+            if (!class_exists(Validation::class)) {
```

Comment:
There was 2 chances :) I'm ok to revert, but fabpot started like this (see https://github.com/symfony/symfony/pull/20189/files) and IMHO it's a good opportunity to make the code more readable